### PR TITLE
Connection errors in gavel error interface

### PR DESCRIPTION
**Pull request by netmilk**

### CODE REVIEW

File: src/transaction-runner.coffee [hunk 1]
```diff
@@ -420,6 +420,8 @@ class TransactionRunner
 
               test.message = message
               test.results = result
+              test.results['general'] = []
```
Comment (kuba-kubula) [inline src/transaction-runner.coffee]: @netmilk `results` or `result`? __blocking__

Comment (netmilk) [inline src/transaction-runner.coffee]: `results`

Comment (netmilk) [inline src/transaction-runner.coffee]: `test.results = result`

File: src/reporters/apiary-reporter.coffee [hunk 2]
```diff
@@ -132,10 +132,16 @@ class ApiaryReporter
       data = @_transformTestToReporter test
       data.result = 'error'
 
+      data['resultData']['result']['general'] ?= []
+
       if CONNECTION_ERRORS.indexOf(error.code) > -1
-        data.resultData.errors.push "Error connecting to server under test!"
+        data['resultData']['result']['general'].push {
```
Comment (kuba-kubula) [inline src/reporters/apiary-reporter.coffee]: Maybe a comment about "using similar interface as Gavel to keep better interoperability" would be nice.

**Pull request merged**